### PR TITLE
deprecate glue_load_plans rake

### PR DESCRIPTION
**Pull request by jacobkagon**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [ ] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/n/projects/2640060/stories/186080689

# A brief description of the changes

Current behavior: Maine does not have plans from the Plan model. DC does but we must be the rake ```bundle exec rake seed:load_products```

New behavior: we must be the rake ```bundle exec rake seed:load_products```

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

**Review by jayreddy519 — DISMISSED** [review]
lgtm.

File: lib/tasks/migrations/plans/glue_load_plans_deprecated.rake [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#! This rake is deprecated. Please use the rake `bundle exec rake seed:load_products`
+
+namespace :seed do
+  task :load_plans, [:year] => :environment do |task, args|
```
Comment (saikumar9) [inline lib/tasks/migrations/plans/glue_load_plans_deprecated.rake:4]: @jacobkagon Can you also rename the task? This way the code will not execute when we run `bundle exec rake seed:load_plans`

**Review by saikumar9 — CHANGES_REQUESTED** [review]

**Review by saikumar9 — APPROVED** [review]
lgtm!

**Review by jayreddy519 — APPROVED** [review]
updates task name && 

updated comment on the file this rake is being deprecated.

**Pull request merged**